### PR TITLE
Add documentation for bulk operations on notification requests

### PR DESCRIPTION
**Pull request by ClearlyClaire**

### CODE REVIEW

File: content/en/methods/notifications.md [hunk 1]
```diff
@@ -827,7 +827,101 @@ Authorization
...
+**OAuth:** User token + `write:notifications`\
+**Version history:**\
+4.3.0 - added
+
+#### Request
+
+##### Form data parameters
+
+:id[]
+: {{<required>}} Array of String. The ID of the notification requests in the database.
```
Comment (oneiros) [inline content/en/methods/notifications.md:866]: Should this not read "ID**s**" (plural)?

File: content/en/methods/notifications.md [hunk 2]
```diff
@@ -827,7 +827,101 @@ Authorization
...
```
Comment (oneiros) [inline content/en/methods/notifications.md:913]: Same as above (IDs, plural).

File: content/en/methods/notifications.md (hunk 1 above)
Comment (ClearlyClaire) [reply content/en/methods/notifications.md:866]:
```suggestion
: {{<required>}} Array of String. The IDs of the notification requests in the database.
```

File: content/en/methods/notifications.md (hunk 2 above)
Comment (ClearlyClaire) [reply content/en/methods/notifications.md:913]:
```suggestion
: {{<required>}} Array of String. The IDs of the notification requests in the database.
```

**Review by oneiros — APPROVED** [review]

**Pull request merged**